libs/components/packages/src/schematics/migrations/update-10/remove-nx-cache/remove-nx-cache.schematic.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { Tree } from '@angular-devkit/schematics';
+import {
+  SchematicTestRunner,
+  UnitTestTree,
+} from '@angular-devkit/schematics/testing';
+
+describe('remove-nx-cache.schematic', () => {
+  it('should run successfully', async () => {
+    const collectionPath = require.resolve('../../migration-collection.json');
+    jest.mock('child_process', () => ({
+      execSync: jest.fn(),
+    }));
+    const runner = new SchematicTestRunner('schematics', collectionPath);
+    const tree = new UnitTestTree(Tree.empty());
+    tree.create('.gitignore', '');
+    tree.create('package.json', '{}');
+    await runner.runSchematic('remove-nx-cache', {}, tree);
+    expect(tree.readText('.gitignore')).not.toContain('.nx/cache');
```

Comment:
I'm confused. Aren't we trying to add `.nx/cache` to the gitignore? Why is it making sure it doesn't contain it?

Reply:
I'd originally implemented this to only add the rule if the `@angular-eslint/build` was a dependency, but I've removed that condition since the rule is benign if it's not needed. This test now asserts that the rule is present regardless of `@angular-eslint/build`.